app/src/main/java/com/mk/tasky/core/presentation/MainActivity.kt

```diff
@@ -203,6 +205,12 @@ fun MainScreen(
         ) {
             val eventTitle = it.savedStateHandle.get<String>("event_title") ?: ""
             val eventDescription = it.savedStateHandle.get<String>("event_description") ?: ""
+            val selectedPhoto = it.savedStateHandle.get<String>("photo_location") ?: ""
```

Comment:
I'd prefer to represent non-existing values as `null` since that's what nullables are used for. If you were new in this project, it wouldn't be obvious that this value is an empty string in case there is no selected photo. 
For the other fields here, you can leave it since an event title can be existent, but empty at the same time.

Reply:
Yeah makes sense. Fixed!